### PR TITLE
Add reading time to reading list

### LINKED ISSUE #2862: Reading List: Add the reading time
**Is your feature request related to a problem? Please describe.**
Reading time for articles are important to be seen in the reading list to manage the time to read, or to just know if I have enough time to read this article on my break.

**Describe the solution you'd like**
Appear the reading time without opening the article.

### PR DESCRIPTION
**Pull request by mariocsee**
## What type of PR is this? (check all applicable)

- [x] Feature

## Description
Adds reading time to reading list

## Related Tickets & Documents
resolves #2862 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
Screen Shot 2019-05-21 at 16 59 34

## Added to documentation?

- [x] no documentation needed

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -685,6 +685,7 @@
     t.float "points", default: 1.0
     t.integer "reactable_id"
     t.string "reactable_type"
+    t.integer "reading_time", default: 0
```
Comment (benhalpern) [inline db/schema.rb]: You shouldn't need to add `reading_time` to the reaction model. You can include it in the Algolia index without it having to be on the reaction have it in its own table—similar to the other models here like `reactable_published_date` etc.

Comment (mariocsee) [reply db/schema.rb]: Removed it from the model and undid the migration.

**Comment by maestromac** [comment]
This looks good but we would need to reindex algolia as soon as this is merged.

File: app/models/reaction.rb [hunk 2]
```diff
@@ -25,7 +25,7 @@ class Reaction < ApplicationRecord
   before_destroy :remove_algolia
 
   algoliasearch index_name: "SecuredReactions_#{Rails.env}", auto_index: false, auto_remove: false do
-    attribute :id, :reactable_user, :searchable_reactable_title, :searchable_reactable_path, :status,
+    attribute :id, :reactable_user, :searchable_reactable_title, :searchable_reactable_path, :status, :reading_time,
               :searchable_reactable_text, :searchable_reactable_tags, :viewable_by, :reactable_tags, :reactable_published_date
     searchableAttributes %i[searchable_reactable_title searchable_reactable_text
                             searchable_reactable_text searchable_reactable_tags reactable_user]
```
Comment (bolariin) [inline app/models/reaction.rb]: It's not related to code you wrote but I noticed `searchable_reactable_text` appears twice.

Comment (benhalpern) [reply app/models/reaction.rb]: Nice catch.

Comment (mariocsee) [reply app/models/reaction.rb]: Just removed the duplicate. Thanks for pointing this out, @bolariin! 😄

**Pull request merged**